modules/dropdown/main/index.coffee

```diff
@@ -135,11 +140,29 @@ class Dropdown extends hx.EventEmitter
     if @isOpen() then @hide(cb) else @show(cb)
     this
 
+  dropdownContent: (dropdownContent, cb) ->
+    if arguments.length
+      setupDropdown = dropdownContentToSetupDropdown dropdownContent
+      @_.dropdownContent = dropdownContent
+      @_ = hx.shallowMerge @_, {
+        setupDropdown,
+        dropdownContent
+      }
```

Comment:
Unless I've misunderstood why we're doing this, you're essentially setting dropdown content twice - we could remove the `shallowMerge` and just set `@_.setupDropdown` if `setupDropdown` is defined?

Also, if you set `setupDropdown` to be invalid, should we still attempt to render?